src/components/Dropdown/components/ValueContainer/ValueContainerHooks.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+
+export function useHiddenOptionsData({ isMultiline, ref, selectedOptionsCount, chipClassName }) {
+  const [overflowIndex, setOverflowIndex] = useState(-1);
```

Comment:
```suggestion

  const [overflowIndex, setOverflowIndex] = useState(-1);
```

Reply:
prefer not to add enter in the first line of a function (its not our standard)